21.js

```diff
@@ -0,0 +1,124 @@
+playerScoreTitle.innerText = "PLAYER SCORE"
+let playerScore = document.createElement("h2")
+let gameMainCompCards = document.createElement("div")
+gameMainCompCards.setAttribute("class", "gameMainCompCards")
+let gameMainCompScore = document.createElement("div")
+gameMainCompScore.setAttribute("class", "gameMainCompScore")
+let compScore = document.createElement("h2")
+let compScoreTitle = document.createElement("h2")
+compScoreTitle.innerText = "DEALER SCORE:"
+let hit = document.createElement("button")
+hit.setAttribute("value", "HIT")
+hit.innerText = "HIT"
+let hitDiv = document.querySelector(".hit")
+let stay = document.createElement("button")
+stay.innerText = "STAY"
+let stayDiv = document.querySelector(".stay")
+
+try { 
+    const fetchData = async (url, callback) => {
+    let res = await axios.get(url)
+    callback(res.data)
+    }
+
+    const setCardValue = (data) => {
+        switch(data) {
+            case "JACK":
+            case "QUEEN":
+            case "KING":
+                return 10
+            case "ACE":
+                if (score >= 13) {
+                    return 1
+                } else {
+                    return 11
+                }
+            default:
+                return Number(data)
+        }
+    }
+
+    const renderPlayerData = (data) => {
+        let card = data["cards"]
+        card.forEach(el => {
+            score += setCardValue(el.value)
+            let img = document.createElement("img")
+            img.src = el.image
+            gameMainPlayerCards.appendChild(img)
+        })
+        playerScore.innerText = score
+        gameMainPlayerScore.appendChild(playerScore)
+    }
+
+    const renderCompData = (data) => {
+        let card = data["cards"]
+        let score = 0
+        card.forEach(el => {
+            score += setCardValue(el.value)
+            let img = document.createElement("img")
+            img.src = el.image
+            gameMainCompCards.appendChild(img)
+        })
+        compScore.innerText = score
+        gameMainCompScore.appendChild(compScore)
+    }
+    
+    fetchData("https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1", (data) => deckId = data.deck_id)
+    
+    start.addEventListener("click", () => {
+        gameMain.innerHTML = ""
+        hitDiv.appendChild(hit)
+        stayDiv.appendChild(stay)
+        fetchData(`https://deckofcardsapi.com/api/deck/${deckId}/draw/?count=2`, renderPlayerData)
+        gameMain.appendChild(gameMainPlayerCards)
+        gameMain.appendChild(gameMainPlayerScore) 
+        gameMainPlayerScore.appendChild(playerScoreTitle)
+    })
+    
+    hit.addEventListener("click", () => {
+        fetchData(`https://deckofcardsapi.com/api/deck/${deckId}/draw/?count=1`, renderPlayerData)
+        if (score > 21) {
+
+        }
+    })
+    
+    stay.addEventListener("click", () => {
+        fetchData(`https://deckofcardsapi.com/api/deck/${deckId}/draw/?count=3`, renderCompData)
```

Comment:
You would need to at the logic here, since you're staying you no longer can increase your score. You can check if your score is greater than the dealer.